src/types.ts

```diff
@@ -19,7 +19,7 @@ export interface PluginOptions {
   maxDepth?: number;
   maxSize?: number;
   strict?: boolean;
-  typeCheck?: boolean | string;
+  typeCheck?: boolean | string | string[];
```

Comment:
glob also supports a string array and when I was playing around having everything go through the type check, I needed the string array in order to get all the tests to pass since their fixtures are in different folders

```js
typeCheck: ['./src/**/*', './types/**/*', './tests/**/*'],
```